pkg/blockstorage/awsefs/conversion.go

```diff
@@ -90,8 +94,9 @@ func snapshotFromRecoveryPointByVault(rp *backup.RecoveryPointByBackupVault, vol
 	if rp.RecoveryPointArn == nil {
 		return nil, errors.New("Recovery point has no RecoveryPointArn")
 	}
-	if volume == nil {
-		return nil, errors.New("Nil volume as argument")
+	encrypted := false
+	if volume != nil {
+		encrypted = volume.Encrypted
 	}
```

Comment:
What happens when the volume is `nil` since we are removing this check?

Reply:
`blockstorage.Snasphot`'s `Volume` field will be `nil`, `Encrypted` field will be `false`. 
Do you see any panic situation?

Reply:
I don't see any usage of `blockstorage.Snapshot.Volume` for EFS but want to make sure it's okay to set it to `nil`